api/context.go

```diff
@@ -46,51 +46,55 @@ type Context struct {
 }
 
 func ApiAppHandler(h func(*Context, http.ResponseWriter, *http.Request)) http.Handler {
-	return &handler{h, false, false, true, false, false, false}
+	return &handler{h, false, false, true, false, false, false, false}
```

Comment:
Next time, lets break this out as named params.  It's getting kind of hard to follow.